src/Tizen.Uix.InputMethod/Interop/Interop.InputMethod.cs

```diff
@@ -219,9 +219,6 @@ public void Dispose()
         [DllImport(Libraries.InputMethod, EntryPoint = "ime_set_dotnet_flag")]
         internal static extern ErrorCode ImeSetDotnetFlag(bool set);
 
-        [DllImport(Libraries.InputMethod, EntryPoint = "ime_set_size")]
-        internal static extern ErrorCode ImeSetSize(int portraitWidth, int portraitHeight, int landscapeWidth, int landscapeHeight);
-
```

Comment:
you could probably remove ImeGetMainWindow() at 202 too

Reply:
I have deleted the ImeGetMainWindow(). thank you.